src/main/scripts/ci/after-success.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ -z "$CODACY_PROJECT_TOKEN" ]; then
+	find -name jacoco.xml -exec java -jar ~/codacy-coverage-reporter-assembly.jar report -l Java -r {} \;
```

Comment:
dont it miss --partial somewhere?

```
Multiple coverage reports for the same language

In order to send multiple reports for the same language, you need to upload each report separately with the flag --partial and then notify Codacy, after all reports were sent, with the final command.

Example

codacy-coverage-reporter report -l Java -r report1.xml --partial
codacy-coverage-reporter report -l Java -r report2.xml --partial
codacy-coverage-reporter final
```
https://github.com/codacy/codacy-coverage-reporter

Reply:
Fixed on https://github.com/scalecube/enable-ci-cd-using-travis/pull/18